java-client/src/test/java/org/opensearch/client/opensearch/_types/QueryCacheStatsTest.java

```diff
@@ -0,0 +1,39 @@
+package org.opensearch.client.opensearch._types;
+
+import static org.junit.Assert.assertEquals;
+
+import jakarta.json.stream.JsonParser;
+import java.io.StringReader;
+import org.junit.Test;
+import org.opensearch.client.json.jackson.JacksonJsonpMapper;
+
+public class QueryCacheStatsTest {
+
+    @Test
+    public void testLongSerailization() {
+        QueryCacheStats expected = new QueryCacheStats.Builder().cacheCount(8757938874l)
+            .cacheSize(8757938874l)
+            .evictions(8757938874l)
+            .hitCount(8757938874l)
+            .memorySizeInBytes(8757938874l)
+            .missCount(8757938874l)
+            .totalCount(8757938874l)
+            .build();
```

Comment:
Lets use different numbers for each, just so we know we're deserializing the right field?